openrtx/src/core/state.c

```diff
@@ -136,7 +142,15 @@ void state_task()
     state.volume = vol / 2;
 
     state.charge = battery_getCharge(state.v_bat);
+    // There is a bug where during spectrum operation, the RSSI reads collide.
```

Comment:
Don't manage this here, do it in rtx.cpp to keep this code agnostic of what is going on